dlrs/clearlinux/releasenote.md

```diff
@@ -1,15 +1,15 @@
-# Deep Learning Reference Stack
+# Deep Learning Reference Stack Release Notes
 
 The Deep Learning Reference Stack, an integrated, highly-performant open source stack optimized for Intel® Xeon® Scalable platforms. This open source community release is part of our effort to ensure AI developers have easy access to all of the features and functionality of the Intel platforms.  The Deep Learning Reference Stack is highly-tuned and built for cloud native environments. With this stack, we are enabling developers to quickly prototype by reducing the complexity associated with integrating multiple software components, while still giving users the flexibility to customize their solutions. This version includes additional components to provide greater flexibility and a more comprehensive take on the deep learning environment.
 
-# The Deep Learning Reference Stack Release
+## The Deep Learning Reference Stack Release
 
 To offer more flexibility, we are releasing multiple versions of the Deep Learning Reference Stack. All versions are built on top of the Clear Linux OS, which is optimized for IA.
 
 > **Note:**
      The minimum validated version of Clear Linux for this stack is 31290.
```

Comment:
Mind updating these versions as in this file [releasenotes.md](https://github.com/intel/stacks/blob/dplascen/dlrs-v0.6.0/dlrs/clearlinux/releasenote.md)?

Reply:
Done with latest commit.  I also bumped some of the headings so the gh-pages site reads cleaner.